diff --git a/dist/stt.cjs b/dist/stt.cjs
index 751d4e807298899837f7af9afca2401491c83bf9..fc49ff81b2450dddd3f5d40daed7ef7e2ccd6977 100644
--- a/dist/stt.cjs
+++ b/dist/stt.cjs
@@ -640,7 +640,7 @@ class SpeechStream extends import_agents.stt.SpeechStream {
     const startTime = words.length > 0 ? ((_a = words[0]) == null ? void 0 : _a.start) ?? 0 : 0;
     const endTime = words.length > 0 ? ((_b = words[words.length - 1]) == null ? void 0 : _b.end) ?? 0 : 0;
     const languageCode = data.language_code ?? this.#language;
-    const normalizedLanguage = languageCode ? (0, import_agents.normalizeLanguage)(languageCode) : (0, import_agents.normalizeLanguage)("en");
+    const normalizedLanguage = languageCode ? (0, import_agents.normalizeLanguage)(languageCode) : (0, import_agents.normalizeLanguage)("");
     const speechData = {
       language: normalizedLanguage,
       text,
diff --git a/dist/stt.js b/dist/stt.js
index 3af2f48a2cac7122cf218af172d46bb92aa95744..d4155d682c3b5fe7673f03fe49497e6b4e87e439 100644
--- a/dist/stt.js
+++ b/dist/stt.js
@@ -634,7 +634,7 @@ class SpeechStream extends stt.SpeechStream {
     const startTime = words.length > 0 ? ((_a = words[0]) == null ? void 0 : _a.start) ?? 0 : 0;
     const endTime = words.length > 0 ? ((_b = words[words.length - 1]) == null ? void 0 : _b.end) ?? 0 : 0;
     const languageCode = data.language_code ?? this.#language;
-    const normalizedLanguage = languageCode ? normalizeLanguage(languageCode) : normalizeLanguage("en");
+    const normalizedLanguage = languageCode ? normalizeLanguage(languageCode) : normalizeLanguage("");
     const speechData = {
       language: normalizedLanguage,
       text,
